Automattic/wp-calypso
# Jetpack plans: new descriptions test

As proposed in`p7kMJG-1pI-p2`, we are going to run a test that changes the descriptions of the features on the jetpack plans pages.

This is how the test looks like:
![image](https://cloud.githubusercontent.com/assets/1554855/24258063/ececa4ec-0fec-11e7-972a-439d162aa739.png)

![image](https://cloud.githubusercontent.com/assets/1554855/24257891/58212860-0fec-11e7-9c9a-55d20b6cfdf9.png)


How to test
====

1. Go to http://calypso.localhost:3000/jetpack/connect/store and check the old plans page style and make sure is still three
2. Open the developer console and run `localStorage.setItem( 'ABTests', '{"jetpackNewDescriptions_20170327":"showNew"}' );`
3. Reload the page
4. The new descriptions & look should be there
5. Go to http://calypso.localhost:3000/ and select a jetpack site. Go to the plans page. You should see the new Style & descriptions there too
